css-view-transitions-2/Overview.bs

```diff
@@ -405,44 +416,57 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 
 		Note: This would wait until a transition is captured or skipped before proceeding to unloading the old document and activating the new one.
 	</div>
 
 	<div algorithm="monkey patch to rendering">
 		Run the following step in [=update the rendering|updating the renedering=], before [=running the animation frame callbacks=]:
 
-		1. For each [=fully active=] {{Document}} |doc| in |docs|, run the [=ready to render=] steps for |doc|.
+		1. For each [=fully active=] {{Document}} |doc| in |docs|, run the [=reveal=] steps for |doc|.
 	</div>
 
 	<div algorithm="monkey patch to reactivation">
 		Run the following step at the end of [=Document/reactivate=]:
 
-		1. Set |document|'s [=ready to render fired=] to false.
+		1. Set |document|'s [=pagereveal fired=] to false.
 	</div>
 
-	<div algorithm="page ready to render">
-	When {{Document}} |document| is <dfn>ready to render</dfn>:
-		1. If |document|'s [=document/ready to render fired=] is true, then return.
+	<div algorithm="page reveal">
+		To <dfn>reveal</dfn> {{Document}} |document|:
+			1. If |document|'s [=document/page pagereveal fired=] is false, then:
+				1. Let |transition| be the result of [=activating cross-document view-transition=] for |document|.
 
-		1. Let |transition| be the result of getting the [=inbound cross-document view-transition=] for |document|.
+				1. [=Fire an event=] named <code>pagereveal</code> at |document|'s [=relevant global object=],
+					using {{PageRevealEvent}}, with [=PageRevealEvent/view transition=] initialized to .
 
-		1. If |transition| is not null and |document| does not [=opt in to cross-document view transitions=], then [=skip the view transition|skip=] |transition| and set |transition| to null.
+				1. Set |document|'s [=document/page pagereveal fired=] to true.
+	</div>
+
+## Setting up and activating the cross-document view transition ## {#setting-up-and-activating-the-cross-document-view-transition}
+
+### Resolving the ''@view-transition''' rule
+
+	<div algorithm>
+		To get the <dfn>resolve @view-transition rule</dfn> for a {{Document}} |document|:
 
-		1. Fire a new event named <code>reveal</code> on |document|'s [=relevant global object=],
-			using {{RevealEvent}}.
+		1. Let |matchingRule| be the last ''@view-transition'' rule in |document| which has a ''@view-transition/navigation'' descriptor whose [=computed value=] is ''@view-transition/navigation/auto''.
 
-		1. If |transition| is not null, then [=activate view transition|activate=] |transition|.
+		1. If |matchingRule| is not found, then return "<code>not found</code>".
 
-		1. Set |document|'s [=document/ready to render fired=] to true.
+		1. If |matchingRule| contains a ''@view-transition/type'' descriptor, then return a [=list=] of strings corresponding to that descriptor's [=computed value=].
```

Comment:
@noamr an empty list vs null has the same effect right? Based on my reading [here](https://drafts.csswg.org/css-view-transitions-2/#ref-for-viewtransition-active-types:~:text=viewTransition%E2%80%99s%20active%20types%20contains%20at%20least%20one%20of%20the%20%3Ccustom%2Dident%3E%20values%20of%20the%20%3Cvt%2Dtype%2Dselector%3E.) it does so just confirming.

Reply:
Yes, though there is no way to specify an empty list.